lib/ohloh_scm/adapters/cvs/misc.rb

```diff
@@ -187,5 +187,13 @@ def ensure_host_key
 			  stdout, stderr = run_with_err(cmd)
       end
     end
+
+    def tags
+      tag_strings = `cvs -Q -d #{ url } rlog -h #{ module_name } | awk -F"[.:]" '/^\\t/&&$(NF-1)!=0'`.split(/\n/)
```

Comment:
Use `run` method to get the system output, which is consistent across the application.